src/main/java/edu/cornell/kfs/fp/service/impl/RecurringDisbursementVoucherDocumentServiceImpl.java

```diff
@@ -409,8 +417,117 @@ public Object call() throws WorkflowException {
             throw new RuntimeException("cancelDVAsSystemUser() Unable to cancel DV: " + dvDocumentNumber, e);
         }
     }
+
+    @Override
+    @SuppressWarnings("unchecked")
```

Comment:
Do we need this?  The rest of the functions in the class aren't suppressing warnings